pkg/controllers/services_controller.go

```
@@ -96,6 +96,83 @@ type ServicesController struct {
+// node hosts the backend pod, and withdraws it otherwise. Both objects must
+// already have been checked with hasValidServiceIP/hasValidEndpointIP.
+func (c *ServicesController) applyRules(svc *v1.Service, ep *v1.Endpoints, ctx string) {
+	svcIP := svc.Status.LoadBalancer.Ingress[0].IP
+	podIP := ep.Subsets[0].Addresses[0].IP
+
+	if !c.servesEndpoint(ep) {
+		c.withdrawRules(svcIP, podIP, ctx+" (backend not on this node)")
+		return
+	}
+
+	c.Proxy.EnsureRules(svcIP, podIP)
```

golangci-lint (2.12.2)

[error] Error return value of `c.Proxy.EnsureRules` is not checked

(errcheck)